src/main/java/io/supertokens/httpRequest/HttpRequest.java

```diff
@@ -152,12 +152,12 @@ public static <T> T sendGETRequestWithResponseHeaders(Main main, String requestI
             if (version != null) {
                 con.setRequestProperty("api-version", version + "");
             }
-
+            con.setInstanceFollowRedirects(false);
```

Comment:
This should be an input param, cause we use this same function in other places too, and we wouldn't want to change the behaviour of those functions.

Reply:
yeah, that's true. For now, it is only used in two tests, for them it makes no difference as far as I can see. On the other hand, this is not the default behaviour of the `HttpURLConnection` so in the future it could be a surprise. I change it to a param.